### DISCUSSION TITLE
Bluetooth?

### DISCUSSION THREAD

**Discussion by alsa32** [fayaaz/mixxx-pi-gen#64] [Q&A]
How do i connect a bluetooth device instead of using 3.5mm ports?
I'm guessing i need pulseaudio for that but it isn't included in the image. Can i just download it with ethernet or do i need to recompile this image with pulseaudio?

**Comment by fayaaz** [comment]
You can try installing pulseaudio or pipewire to get bluetooth working.
The below link should work to install them as the image is just a modified version of the Raspberry Pi OS (which itself is a modified debian)
https://forums.raspberrypi.com/viewtopic.php?t=62851

I'm afraid after that you'll have to work out how to get it all playing nicely together. For me, adding bluetooth or pulseaudio would add too much (unacceptable) latency but YMMV.
